graphqlapi/src/main/java/io/stargate/graphql/web/resources/FilesResource.java

```diff
@@ -98,33 +101,51 @@ public Response getSchema(
       throws Exception {
 
     if (!DmlResource.KEYSPACE_NAME_PATTERN.matcher(keyspace).matches()) {
-      LOG.warn("Invalid keyspace in URI, this could be an XSS attack: {}", keyspace);
+      LOG.warn("Malformed keyspace in URI, this could be an XSS attack: {}", keyspace);
       // Do not reflect back the value
-      return Response.status(Response.Status.BAD_REQUEST).entity("Invalid keyspace name").build();
-    }
-
-    if (!isAuthorized(token, httpRequest)) {
-      return Response.status(Response.Status.UNAUTHORIZED).build();
+      return Response.status(Response.Status.BAD_REQUEST).entity("Malformed keyspace name").build();
     }
 
     UUID versionUuid = null;
     if (version != null) {
       try {
         versionUuid = UUID.fromString(version);
       } catch (IllegalArgumentException e) {
-        return notFound(keyspace, version);
+        LOG.warn("Malformed version in URI, this could be an XSS attack: {}", version);
+        return Response.status(Response.Status.BAD_REQUEST).entity("Malformed version").build();
```

Comment:
This is only tangentially related, but as I moved this check to the top of the method I realized it should be consistent with the keyspace one.